### DISCUSSION TITLE
Why no script for Windows VM?

### DISCUSSION THREAD

**Discussion by yuriw** [tteck/Proxmox#2632] [General]
Hello 
I wonder if there are any licensing limitations on why there is no a script for auto installation and configuration of windows vm?
TIA

**Comment by OddSquirrel** [comment]
I wouldn't be surprised if licensing made it more complicated than it has to be.

On the other hand, it's so easy to set up a template once and just create a VM when needed. That's what I do whenever I need a quick Windows VM to try something out, and it works a treat.
